scripts/engine/V3/generic-engine-deployer.ts

```diff
@@ -207,13 +208,21 @@ async function main() {
     await randomizer.deployed();
     const randomizerAddress = randomizer.address;
     console.log(
-      `[INFO] Randomizer ${deployDetails.randomizerContractName} deployed at ${randomizerAddress}`
+      `[INFO] Randomizer ${deployDetails.randomizerContractName} deployed at g${randomizerAddress}`
     );
     await delay(EXTRA_DELAY_BETWEEN_TX);
 
     // Deploy Core contract
+    // Ensure that BytecodeStorageReader library is linked in the process
+    const bytecodeStorageLibraryAddress =
+      BYTECODE_STORAGE_READER_LIBRARY_ADDRESSES[networkName];
```

Comment:
can you add a check in the validation section to catch if this is undefined before we perform any deployment actions? (e.g. immediately fail if we were to run this today on arbitrum-goerli)

Reply:
Great call – done!